wormhole-connect/src/utils/address.ts

```diff
@@ -0,0 +1,60 @@
+import { Chain, chainToPlatform, Wormhole } from '@wormhole-foundation/sdk';
+import { isValidSuiAddress } from '@mysten/sui.js';
+import { PublicKey } from '@solana/web3.js';
+import { getAddress } from 'ethers';
+
+function isEvmAddress(address: string): boolean {
+  if (!/^0x[0-9a-fA-F]{40}$/.test(address)) return false;
+  try {
+    getAddress(address);
+    return true;
+  } catch {
+    return false;
+  }
+}
+
+function isSolanaAddress(address: string): boolean {
+  try {
+    new PublicKey(address);
+    return true;
+  } catch {
+    return false;
+  }
+}
+
+function isAptosAddress(address: string): boolean {
+  return /^0x[0-9a-fA-F]{64}$/.test(address);
+}
+
+export function isValidWalletAddress(chain: Chain, address: string): boolean {
```

Comment:
I suppose this function could belong in the SDK, but adding it here was easier and it's sort of specific to how we want to handle wallet input addresses in Connect. I will move it if that would be better though